os/getting-started-with-systemd.md

```diff
@@ -8,7 +8,7 @@ systemd consists of two main concepts: a unit and a target. A unit is a configur
 
 systemd is the first process started on CoreOS and it reads different targets and starts the processes specified which allows the operating system to start. The target that you'll interact with is the `multi-user.target` which holds all of the general use unit files for our containers.
 
-Each target is actually a collection of symlinks to our unit files. This is specified in the unit file by `WantedBy=multi-user.target`. Running `systemctl enable foo.service` creates symlinks to the unit inside `multi-user.target.wants`.
+Each target is actually a collection of symlinks to our unit files. This is specified in the unit file by `WantedBy=multi-user.target`. Running `systemctl enable foo.service` creates symlinks to the unit inside `multi-user.target.wants`. Please note that `systemctl enable foo.service` may affect the boot process by race condition exception in network configuration when [Cloud-Config] configures network according to cloud provider's `meta_data` and `user_data`. For the common usage you have to put your unit files inside Cloud-Config's [units][cloud-config-units] section.
```

Comment:
I don't like this advice. I think we should tell our users that if they want to use cloud-init, all services should be `runtime: true` and `enabled: false`. This way, every boot is the same.

There is nothing wrong with systemd enable, it is entirely our fault. When people start moving to Ignition, they will use systemd enable to run there services and it will work perfectly. I don't want to mislead people here.